src/Process.js

```diff
@@ -18,24 +17,17 @@ import type {SpawnOptions} from './types';
  * @returns {ChildProcess}
  */
 // eslint-disable-next-line import/prefer-default-export
-export const createProcess = (
-  workspace: ProjectWorkspace,
-  args: Array<string>,
-  options?: SpawnOptions = {}
-): ChildProcess => {
-  // A command could look like `npm run test`, which we cannot use as a command
-  // as they can only be the first command, so take out the command, and add
-  // any other bits into the args
-  const runtimeExecutable = workspace.pathToJest;
-  const parameters = runtimeExecutable.split(' ');
-  const command = parameters[0];
-  const initialArgs = parameters.slice(1);
-  const runtimeArgs = [].concat(initialArgs, args);
+export const createProcess = (workspace: ProjectWorkspace, args: Array<string>): ChildProcess => {
+  let runtimeExecutable = workspace.pathToJest;
+
+  if (args.length) {
+    runtimeExecutable += ` ${args.join(' ')}`;
+  }
 
   // If a path to configuration file was defined, push it to runtimeArgs
   if (workspace.pathToConfig) {
-    runtimeArgs.push('--config');
-    runtimeArgs.push(workspace.pathToConfig);
+    runtimeExecutable += ' --config ';
```

Comment:
while using string `+=` worked, one has to be mindful about adding space separators manually. It might be simpler to just push all command and arguments into an array, then use the `array.join(' ')` to construct the final command line?

Reply:
thanks for the suggestion